src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -366,6 +379,31 @@ private void validateName(final String name, final EventTypeBase eventType) thro
         }
     }
 
+    private void validateAuthorization(final EventTypeBase eventType) throws InvalidEventTypeException,
```

Comment:
Well, we have it everywhere in this class. Also, EventTypeBase is not exactly a view object.